samples/k8s-connected-and-managed-cluster/README.md

```diff
@@ -0,0 +1,155 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Connected and Managed Cluster sample
+
+This folder contains samples that show how an ISV can prepare a Kubernetes application that can be installed on Azure Kubernetes Service (AKS) clusters or Azure Arc-enabled Kubernetes Clusters.
+
+Note:
+  - Throughout this sample:
+      - The terms  "connectedClusters" and "connected clusters" denote Azure Arc-enabled Kubernetes clusters.
+      - The terms  "managedClusters" and "managed clusters" denote Azure Kubernetes Service (AKS) clusters.      
+
+## Artifact changes
+
+The following changes are required to enable support for connected and managed clusters.
+
+### Manifest
+
+Add 'supportedClusterTypes' under extensionRegistrationParameters.
+
+Sample below indicates all distributions of connectedClusters type and all ditributions of managedClusters type are supported.
+
+![Alt text](images/Manifest_changes.PNG)
+
+Following is the guidance on the supportedClusterTypes:
+
+| Property      | Description |
+| -----------   | ----------- |
+| supportedClusterTypes       | Contains an object for each top-level cluster-type. Allowed types are – “managedClusters”, “connectedClusters”. "managedClusters" denotes Azure Kubernetes Service (AKS) clusters. "connectedClusters" denotes Azure Arc-enabled Kubernetes clusters. For each of these cluster types, specify distributions and unsupported Kubernetes versions for these distributions. If supportedClusterTypes is not provided, all distributions of ‘managedClusters’ will be supported by default. If supportedClusterTypes is provided, and a given top level cluster type is not provided, then all distributions and Kubernetes versions for that cluster type will be treated as unsupported. |
+| distribution   | An array of distributions  corresponding to the cluster type. Provide name(s) of specific distributions. Set the value to [“All”]  to indicate all distributions are supported. |
+| distributionSupported  | A boolean value representing whether the specified distribution(s) are supported. If false, providing a value for UnsupportedVersions will cause an error. |
+| unsupportedVersions  | A list of versions for the specified distribution(s) which are unsupported. Supported operators: |
+
+                        •	"=" Given version is not supported. E.g.: “=1.2.12”
+
+                        •	">" All versions greater than the given version are not supported. E.g.: “>1.1.13”
+
+                        •	"<" All versions less than the given version are not supported. E.g.: “<1.3.14”
+
+                        •	"..." All versions in range are unsupported. E.g.: ” 1.1.2...1.1.15”  (includes right-side value and excludes left-side value)
```

Comment:
Is this list supposed to be inside the table of unsupported versions?

Reply:
Yes, this is supposed to be inside the previous cell, but table does not allow multi-lines. I added some text that indicates this list is linked to the previous row.